generators/server/index.js

```diff
@@ -902,6 +903,7 @@ module.exports = JhipsterServerGenerator.extend({
             this.copy('gitattributes', '.gitattributes');
             this.copy('editorconfig', '.editorconfig');
             this.template('_travis.yml', '.travis.yml', this, {});
+            this.copy(JHIPSTER_CONFIG_DIR + 'upgrade.sh', JHIPSTER_CONFIG_DIR + 'upgrade.sh');
         },
```

Comment:
@lordlothar99 can you remove the unwanted files and changes related to the sh file?